pkg/taskservice/mysql_task_storage.go

```diff
@@ -0,0 +1,580 @@
+                           task_metadata_id,
+                           task_metadata_executor,
+                           task_metadata_context,
+                           task_max_retry_times,
+                           task_retry_interval,
+                           task_delay_duration,
+                           task_parent_id,
+                           task_status,
+                           task_runner,
+                           task_epoch,
+                           last_heartbeat,
+                           create_at,
+                           end_at
+					) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+
+	updateAsyncTask = `update sys_async_task set 
+							task_metadata_executor=?,
+							task_metadata_context=?,
+							task_max_retry_times=?,
+							task_retry_interval=?,
+							task_delay_duration=?,
+							task_parent_id=?,
+							task_status=?,
+							task_runner=?,
+							task_epoch=?,
+							last_heartbeat=?,
+							result_code=?,
+							error_msg=?,
+							create_at=?,
+							end_at=? where task_id=?`
+
+	selectAsyncTask = `select 
+    						task_id,
+							task_metadata_id,
+							task_metadata_executor,
+							task_metadata_context,
+							task_max_retry_times,
+							task_retry_interval,
+							task_delay_duration,
+							task_parent_id,
+							task_status,
+							task_runner,
+							task_epoch,
+							last_heartbeat,
+							result_code,
+							error_msg,
+							create_at,
+							end_at 
+						from sys_async_task`
+
+	insertCronTask = `insert into sys_cron_task (
+                           task_metadata_id,
+						   task_metadata_executor,
+                           task_metadata_context,
+                           task_max_retry_times,
+                           task_retry_interval,
+                           task_delay_duration,
+                           cron_expr,
+                           next_time,
+                           trigger_times,
+                           create_at,
+                           update_at
+                    ) values (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)`
+
+	selectCronTask = `select 
+    						cron_task_id,
+    						task_metadata_id,
+    						task_metadata_executor,
+    						task_metadata_context,
+    						task_max_retry_times,
+    						task_retry_interval,
+    						task_delay_duration,
+    						cron_expr,
+    						next_time,
+    						trigger_times,
+    						create_at,
+    						update_at
+						from sys_cron_task`
+
+	updateCronTask = `update sys_cron_task set 
+							task_metadata_executor=?,
+    						task_metadata_context=?,
+    						task_max_retry_times=?,
+    						task_retry_interval=?,
+    						task_delay_duration=?,
+    						cron_expr=?,
+    						next_time=?,
+    						trigger_times=?,
+    						create_at=?,
+    						update_at=? where cron_task_id=?`
+)
+
+// used for testing
+type mysqlTaskStorage struct {
+	db *sql.DB
+
+	// Used for testing. Make some changes to the data before updating.
```

Comment:
it's not needed in mysql based task storage

Reply:
removed.